app/pages/patients/patients.js

```diff
@@ -98,7 +94,7 @@ export let Patients = React.createClass({
     var handleClickYes = function(e) {
       e.preventDefault();
       self.props.onHideWelcomeSetup();
```

Comment:
Not sure I follow this. I'd have thought you'd def want to clear the message here.